src/normLhalf.jl

```diff
@@ -0,0 +1,54 @@
+# Lhalf pseudo-norm (times a constant)
+
+export RootNormLhalf
+
+"""
+**``L_1/2^(1/2)`` pseudo-norm**
+    RootNormLhalf(λ=1)
+Returns the function
+```math
+f(x) = λ\\cdot \\sum |x|\\^{1/2}
+```
+for a nonnegative parameter `λ`.
+"""
+struct RootNormLhalf{R <: Real} <: ProximableFunction
```

Comment:
there are no shifts here right? This should go to ProximalOperators then.

Reply:
Yep that's fine with me if one of use wants to send a PR or something? Not sure how that works or what the etiquette is.

Reply:
I'm shy though. Should I send an email or just github?

Reply:
ok, let's leave it here for now

Reply:
hahaha alright i guess i can add it once it works